sdks/python/.coveragerc

```diff
@@ -16,4 +16,23 @@
 #
 
 [run]
-omit = target/*
\ No newline at end of file
+omit = target/*
+
+[report]
+exclude_lines =
+  # Have to re-enable the standard pragma
+  pragma: no cover
+  abc.abstractmethod
+
+  # Overload stubs should never be executed.
+  @overload
+
+  # Don't complain about missing debug-only code:
+  def __repr__
+  if self\.debug
```

Comment:
how about: `if self\.debug_logging_enabled` to avoid pattern-matching with `if self.debug_options` which occurs in the codebase?

Reply:
Sure. The reason I did if self\.debug was to make .coveragerc consistent with setup.cfg coverage option. For the infra pr I can make them both self\.debug_logging_enabled